### PR TITLE
New installer version

### LINKED ISSUE #59: Install Script Module Version Missmatch
**Describe the bug**
The install script does not handle the different module versions correctly.

**Expected behavior**
The install script should always take the same version of all modules in order to be persistent and that every module can work correctly.

### LINKED ISSUE #60: Install Script Multiple Mgmt Interfaces
**Describe the bug**
The install script does not allow to configure multiple mgmt interfaces.

**Expected behavior**
There should be a possibility to give a list of mgmt interfaces to the script, which then are configured correctly.

### PR DESCRIPTION
**Pull request by jklaiber**
* Created a new installer version which is simpler and more reliable
* Removed the need for a configuration file and changed the configuration folder from `/etc/wemulate` to `/var/lib/wemulate`

Closes #59 
Closes #60

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -102,152 +78,19 @@ elevate_priv() {
...
-for directory in $conf_folder/*; do
-    bash \$directory/bridge.conf
-    bash \$directory/tc.conf
-done
-EOF
-  completed "Startup configuration $path is generated"
-  $sudo bash -c "cat >> "${cron_config_file}"" << EOF
-@reboot root    bash $path >> $cron_config_file
+  sudo bash -c "cat >> "${cron_config_file}"" << EOF
+@reboot root    wemulate restore device >> $cron_config_file
```
Comment (severindellsperger) [inline install/install.sh:85]: The wemulate restore command is not implemented yet, so we have to stick with the current solution. Can you change that?

File: wemulate/ext/settings/device.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
-from typing import List, Optional
-
 import netifaces
-
 import wemulate.core.database.utils as dbutils
+
+from typing import List, Optional
```
Comment (severindellsperger) [inline wemulate/ext/settings/device.py:4]: Please add here a blank line to differentiate between internal and external dependencies to stick to our new import technique.

File: install/install.sh (hunk 1 above)
Comment (jklaiber) [reply install/install.sh]: Done

File: wemulate/ext/settings/device.py (hunk 2 above)
Comment (jklaiber) [reply wemulate/ext/settings/device.py:4]: Done

**Comment by jklaiber** [comment]
> Is it possible to update the documentation as well? #71 could be done in the same cycle, couldn't it?

Will be done with an additional PR

**Comment by severindellsperger** [comment]
LGMT now.

**Pull request merged**